### PR TITLE
Finished Let's connect Section

### PR DESCRIPTION
**Pull request by MariaTeo**

### CODE REVIEW

File: src/_component/about-me/about-me.scss [hunk 1]
```diff
@@ -37,16 +37,16 @@ button.pill i.icon__download {
 }
 
 button[class^="pill"] i[class^="icon__"]{
```
Comment (RalucaRbc) [inline src/_component/about-me/about-me.scss:39]: I was suggested to have an empty space just after `]` , again, not a big issue at all ✌️

File: src/_component/about-me/about-me.scss [hunk 2]
```diff
@@ -37,16 +37,16 @@ button.pill i.icon__download {
...
-  height: 16px;
-  margin-right: 12px;
+  width: 12px;
+  height: 12px;
   display: flex;
   align-items: center;
   justify-content: center;
 }
 
 .pill--blue{
```
Comment (RalucaRbc) [inline src/_component/about-me/about-me.scss:47]: And another empty space after `.pill--blue`

File: src/_component/connect/connect.scss [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+.connect__avatar{
+  padding-top: 100%;
+  background-color: white;
+  position: relative;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.25);
+  border-radius: 8px;
+  transform: rotate(-2.5deg);
+
```
Comment (RalucaRbc) [inline src/_component/connect/connect.scss:108]: One empty line instead of 2, please don't hate me Maria ❤️

**Review by RalucaRbc — APPROVED** [review]

File: src/_component/about-me/about-me.scss (hunk 1 above)
Comment (MariaTeo) [reply src/_component/about-me/about-me.scss:39]: thank you 💯

File: src/_component/connect/connect.scss (hunk 3 above)
Comment (MariaTeo) [reply src/_component/connect/connect.scss:108]: hahah thanks <3

**Review by psyhedeliq — APPROVED** [review]

**Pull request merged**